### PR TITLE
New Clear and Share bucket Buttons

### PR DESCRIPTION
**Pull request by mwernimont**
Took the clear and share options out of the bucket dropdown and created their own buttons.

@isuftin There is some left over code from the dropdown that I was trying to remove since it doesn't get used anymore, but every time I tried to remove something the app crashed.  Figured a pro could point out what needs to be removed and what needs to stay?

### CODE REVIEW

**Comment by mwernimont** [comment]
Added the new steps to the intro.js and made the mobile dots match the desktop, being only 3

**Comment by isuftin** [comment]
@mwernimont Are those two buttons supposed to be working? They don't work on my end.

File: coastal-hazards-portal/src/main/webapp/js/cch/objects/widget/BucketSlide.js [hunk 1]
```diff
@@ -33,9 +33,11 @@ CCH.Objects.Widget.BucketSlide = function (args) {
 	me.$SLIDE_CONTROLSET = me.$SLIDE_CONTAINER.find('> div > div:first-child');
 	me.$DROPDOWN_CONTAINER = me.$SLIDE_CONTROLSET.find('> div > div:nth-child(2)');
 	me.SLIDE_CONTENT_ID = me.$SLIDE_CONTAINER.find(' .application-slide-content').attr('id');
-	me.$CLOSE_BUTTON = me.$SLIDE_CONTAINER.find('> div > div.application-slide-controlset > div > div:first-child > i');
-	me.$TOP_LEVEL_DROPDOWN_TRIGGER = me.$DROPDOWN_CONTAINER.find('button:first-child');
-	me.$TOP_LEVEL_LIST = me.$DROPDOWN_CONTAINER.find('ul');
+	me.$CLOSE_BUTTON = me.$SLIDE_CONTAINER.find('> div > div.application-slide-controlset > div > div:nth-child(1) > button');
+        me.$CLEAR_BUTTON = me.$SLIDE_CONTAINER.find('> div > div.application-slide-controlset > div > div:nth-child(2) > #clear-bucket');
```
Comment (isuftin) [inline coastal-hazards-portal/src/main/webapp/js/cch/objects/widget/BucketSlide.js]: Please use tabs and not spaces.

**Comment by isuftin** [comment]
Scratch that, looks like it's working. 

What did you want to remove that's breaking the app? What's the old code you're looking to remove?

**Comment by mwernimont** [comment]
All working on my end and doesn't look like theres anything else thats unneeded

**Pull request merged**